lane.py

```diff
@@ -1806,7 +1802,7 @@ def works_in_window(self, _db, facets, target_size):
         )
 
         lane_query = lane_query.order_by(
-            "quality_tier desc", work_model.random.desc()
+            text("quality_tier desc"), work_model.random.desc()
```

Comment:
Without this change, the `default` change caused some test failures -- see https://travis-ci.com/NYPL-Simplified/server_core/jobs/183150692 for an example. The error message was:

`CompileError: Can't resolve label reference for ORDER BY / GROUP BY. Textual SQL expression 'quality_tier desc' should be explicitly declared as text('quality_tier desc')`

I couldn't figure out why this was happening, especially since the tests didn't fail in isolation -- only when you ran the whole suite. But the explicit declaration seemed like good advice anyway.